dist/editor/index.html

```diff
@@ -23,19 +23,9 @@
     <script src="jquery.min.js"></script>
     <!--{endif}-->
 
-    <!-- Lacking browser support -->
-    <script nomodule="" src="redirect-on-no-module-support.js"></script>
-    <script type="module" src="redirect-on-lacking-support.js"></script>
-
```

Comment:
I had these as separate scripts so that:

1. The user whose browser did not support would quickly be notified without need to load the whole app.
2. It wouldn't pollute the main code. While I normally believe in modularity, for old-browser polyfills and such detection, I think it does make more sense to have separate scripts since one can review browser support in one place--i.e., if one is updating browser targets, one can just look in the HTML and drop all polyfills that are no longer needed without a need to dig into the JavaScript files and modify them based on a concern unrelated to a change in behavior in those scripts (though it is still good to add comments indicating the files where the polyfills are needed). Admittedly this could be mitigated if there is an app entrance file separate from the main editor file which handles the importing of polyfills all in one place, but there'd still be problem no. 1.

Reply:
still need to check thos